### ISSUE TITLE
Increase to number of Logical switches

### ISSUE THREAD

**Issue by Woita2095** [FrSky-OS/HORUS-X10-X10S-X12S-bug-report#176]
Please increse number of Logical switches.
LS seems to be very strong tool, with their new functionality (delay, etc).
8 of LS is too little.
It would worth to increse number of Logical switches to 32 or (better) 64.

**Comment by janekx** [comment]
Yes I think so.
If we wanted to program more complec behave we find that LSW are not much we need

**Comment by MIKE-DAILEY** [comment]
I agree more logic switches would be great.

**Comment by ssray0** [comment]
Yes, this would give more capability!

**Comment by Lombardoa** [comment]
I agree, and I would be very happy to see two more pages of LSW in the next update.

**Comment by Maverick440** [comment]
Yes we need more logic switches, I've already used all 8 for telemetery information and am now limited to what I can do with my X10S.
At least 16 would be good  with 32 being even better
